homeassistant/components/konnected/panel.py

```diff
@@ -63,9 +63,8 @@ def __init__(self, hass, config_entry):
         """Initialize the Konnected device."""
         self.hass = hass
         self.config_entry = config_entry
-        self.config = (
-            config_entry.data
-        )  # a configuration.yaml device config contained in a device entry
+        self.config = config_entry.data
+        self.options = config_entry.options
```

Comment:
This will only work if a new panel is always created when the entry data or options changes, since those are replaced when an entry is updated.

Reply:
Yea - that should be how it works.  async_setup_entry creates an update listener which reloads the config entry upon an update.  That will create a replacement panel instance.